Update travis.yml with PHP 7.1

DonCallisto opened the pull request:
Updated travis.yml to test codebase against PHP 7.1

eXorus commented on .travis.yml:
```diff
@@ -5,9 +5,15 @@ php:
+    - 7.1
 
 matrix:
+    include:
```
Could you explain this part ?

DonCallisto commented:
@eXorus for sure

`include` is an explicit statement used to combine something (language, environment, etc) in your build ([https://docs.travis-ci.com/user/customizing-the-build#Explicitly-Including-Jobs](https://docs.travis-ci.com/user/customizing-the-build#Explicitly-Including-Jobs))

As `hhvm` is removed from ubuntu official repo, when need to specify to take it from elsewhere (`trusty` dist has it). So, in order to reproduce old builds behavior, I've added explicitly to `include` matrix with two different kinds of build. I also removed it from `php` entry as, otherwise, travis would have tried to build an environment from its default ubuntu dist (`precise`)

Is this what you would like to know?

eXorus commented:
Ok thanks
